frontend/src/components/CustomElements.js

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { Button, Checkbox, Text, Image, Flex } from "@chakra-ui/core";
+
+const MarginWidest = "45px";
```

Comment:
Try using `rem` for spacing, this is usually much more flexible approach

Reply:
Also what's the point of this const?

Reply:
cuuuz first there were more places i used this and i thought it was not nice to hardcode it x times, now it's only one element that uses this but  i'm totally sure i will use it during next tasks.

Reply:
You should use css vars in situations like that